contrib/single-node.sh

```diff
@@ -25,10 +25,12 @@ gaiad gentx validator 5000000000stake --keyring-backend="test" --chain-id $CHAIN
 gaiad collect-gentxs
 
 # Set proper defaults and change ports
-sed -i 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' ~/.gaia/config/config.toml
-sed -i 's/timeout_commit = "5s"/timeout_commit = "1s"/g' ~/.gaia/config/config.toml
-sed -i 's/timeout_propose = "3s"/timeout_propose = "1s"/g' ~/.gaia/config/config.toml
-sed -i 's/index_all_keys = false/index_all_keys = true/g' ~/.gaia/config/config.toml
+echo 1
```

Comment:
Nit: Should this echo something more meaningful, or maybe not echo at all?

```suggestion
echo "Setting rpc listen address"
```